Src/AutoFoq/AutoFoqCustomization.fs

```diff
@@ -18,7 +18,8 @@ type AutoFoqCustomization(relay: ISpecimenBuilder) =
         AutoFoqCustomization(
             FilteringSpecimenBuilder(
                 MethodInvoker(
-                    FoqMethodQuery()),
+                    FoqMethodQuery(
+                        Fixture())),

```

Comment:
Agreed, let's do it. After all, since this PR targets the `v4` branch things are a bit more flexible.